lasagne/tests/layers/test_conv.py

```diff
@@ -286,9 +298,9 @@ def test_pad(self, DummyInputLayer):
         input_layer = DummyInputLayer((1, 2, 3, 3))
         with pytest.raises(RuntimeError) as exc:
             layer = Conv2DDNNLayer(input_layer, num_filters=1,
-                                   filter_size=(3, 3), border_mode='valid',
+                                   filter_size=(3, 3),
                                    pad=(1, 1))
-        assert ("You cannot specify both 'border_mode' and 'pad'" in
+        assert ("You cannot specify both 'pad' and 'pad'" in
                 exc.value.args[0])
```

Comment:
Uhm, are you sure you executed the tests locally? This is nonsense :) You should just remove the last 7 lines.